public/lessons/server_side_javascript/hello_node.md

```diff
@@ -66,6 +54,23 @@ Wrote to C:\Users\alex\code\hello_node\package.json:
 }
 ```
 
+# Hello, Start!
+
+You will need to add one line to the generated `package.json` file.
```

Comment:
really? that is possibly worse than creating a Procfile

Reply:
No, I don't think that it is. How is it worse?

How many questions came up about the Procfile during the dry run class?

We covered: 
- Why there is no file extension
- Why the syntax is different
- Why Heroku was throwing errors because of missing meaningful whitespace.
- How to read the errors to find the problem (missing an 'npm start' script)

Like I mentioned before, they are both a tradeoff. But since we are having students write Javascript, and read JSON, it may be less of a jump to add a single line of JSON to to output of the `package.json` file.

Reply:
It's worse because "create an empty file and put this one line in it" is much less error-prone than "open this file that magically appeared, then find a spot in it then click your mouse on the **previous** line (on the right side, not where any characters are) then hit return and *then* type this stuff in and don't forget the close quote and comma.

Reply:
The only in-class issue with the Procfile was that one student created it but then skipped the instruction to put text in it.

Reply:
Maybe I remember it differently.

- Bennie asked about whether the file needed to be capitalized, and what extension should be at the end. (conversation about relation to Makefile and history took 5 minutes)
- Alex created the file with the correct capitalization, but it was empty, then after putting the line `web:node index.js` within the file we discovered that the whitespace was incorrect, and it was changed to `web: node index.js`. (correction and debugging of deploy took 10 minutes)

So I'll fall back to my previous points:

- Procfile is weird and different than any other file within the project.
- Students are already learning to write Javascript and JSON, this continues after day one.
- They are both tradeoffs. Procfile syntax is simple but weird. package.json syntax is more familiar but more complicated.

When we spoke about this over Slack you agreed, now you don't. So let's just pick one approach and have it be that way for the course. NPM is the "standard" way to run an application, test, or package, so that's why I thought this would be more consistent and easier to learn. If you disagree than I'll close the PR.

Reply:
On slack I didn't realize the "start" line wasn't already in there. And here on github you've convinced me! I should not fear package.json.

Another variant: maybe we go back to *not* using `npm init` and have them create a `package.json` with two lines instead of one.